android/src/main/java/com/revenuecat/purchases_flutter/PurchasesFlutterPlugin.java

```diff
@@ -342,7 +350,8 @@ private void setupPurchases(String apiKey, String appUserID, @Nullable Boolean o
     private void setUpdatedCustomerInfoListener() {
         Purchases.getSharedInstance().setUpdatedCustomerInfoListener(customerInfo -> {
             if (channel != null) {
-                channel.invokeMethod(CUSTOMER_INFO_UPDATED, CustomerInfoMapperKt.map(customerInfo));
+                Map<String, Object> customerInfoMap = CustomerInfoMapperKt.map(customerInfo);
+                runOnUiThread(() -> channel.invokeMethod(CUSTOMER_INFO_UPDATED, customerInfoMap));
```

Comment:
This is a behavior change, so we should mention it somewhere in the changelog maybe? (same for the ones in iOS)

Reply:
sure. In any case, I was getting crashes when sending the other event not in the UI thread, so I think this would actually be a bug fix more than a behavior change

Reply:
I will extract this to a different PR so it's clear